pandas/tests/series/indexing/test_indexing.py

```diff
@@ -391,6 +391,22 @@ def test_setslice(datetime_series):
     assert sl.index.is_unique is True
 
 
+def test_incorrect_assigment():
```

Comment:
can you change this test name to be more informative, something like

test_error_raised_on_2d_to_1d

Reply:
Thanks for the feedback @jreback! Changed the test name to `test_2d_to_1d_assignment_error_raised`.